videointelligence/annotate/object_tracking.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package annotate contains speech examples.
+package annotate
+
+// [START videointelligence_object_tracking]
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+
+	"context"
+
+	"github.com/golang/protobuf/ptypes"
+
+	video "cloud.google.com/go/videointelligence/apiv1"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+)
+
+func objectTracking(w io.Writer, filename string) error {
```

Comment:
Might want to add function comments saying what each function does. Sometimes, it might be repetitive with the surrounding documentation, though. So, it's up to you.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Sure, added a one-line small description to all functions